.github/workflows/ci.yml

```diff
@@ -109,3 +109,30 @@ jobs:
       - name: Run tests
         run: pnpm ember try:one ${{ matrix.ember-try-scenario }} --skip-cleanup
         working-directory: test-app
+
+  typescript-compatibility:
+    name: Type checking - ${{ matrix.typescript-scenario }}
+    runs-on: ubuntu-latest
+
+    needs: [test]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        typescript-scenario:
+          - typescript@5.0
```

Comment:
Since we added type-checking on the CI based on different TS versions, we are able to inform about version compatibility. Could you add a mention in the [compatibility section](https://github.com/qonto/ember-prismic-dom/tree/addon-typescript-conversion#compatibility) of the README? Something like this:

```md
- TypeScript v5 or above
```

Reply:
Updated, thanks!